R/variables.R

```diff
@@ -178,12 +178,15 @@ Probabilities.SupportVectorMachine <- function(x)
 #' @param object A \code{GradientBoost} object.
 #' @param newdata Optionally, a data frame including the variables used to fit the model.
 #' If omitted, the \code{data} supplied to \code{GradientBoost()} is used before any filtering.
+#' @param keep.soft.probs A logical to control whether output returns soft probabilities
+#' (i.e. probabilities for each categorical level) are kept or output is coerced to a factor
+#' via the largest probability.
 #' @param ... Additional arguments to pass to predict.xgb.Booster.
 #' @importFrom stats complete.cases
 #' @importFrom flipData CheckPredictionVariables
 #' @importFrom xgboost xgb.load.raw
```

Comment:
Add version requirement for `xgboost` to DESCRIPTION corresponding to when this function was added.

Reply:
Updated in https://github.com/Displayr/flipMultivariates/pull/8/commits/c3c307e859165f64c38c1406816e5602d2e09e5a